Backpack/Theme/Classes/BPKThemeContainerController.m

```diff
@@ -51,6 +48,10 @@ - (instancetype)initWithThemeDefinition:(id<BPKThemeDefinition>)themeDefinition
     return self;
 }
 
+- (instancetype)createIdenticalContainerForRootController:(UIViewController *)rootController {
```

Comment:
Since this has been renamed maybe we should clarify the name a bit by making it 
```objective-c
- (instancetype)createIdenticalContainerControllerForRootController:(UIViewController *)rootController;
```